libs/interpreter-lib/README.md

```diff
@@ -0,0 +1,7 @@
+<!--
+SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+
+SPDX-License-Identifier: AGPL-3.0-only
+-->
+
+# Jayvee interpreter Library
```

Comment:
This could be expanded a bit, maybe even with a minimal mermaidjs diagram that shows how the CLI/the interpreter work together.

Reply:
Done in [43b6910](https://github.com/jvalue/jayvee/pull/388/commits/43b6910fd787c25947adab4ac9e5e8b75df70750)